integ-test/src/test/java/org/opensearch/sql/sql/SystemFunctionIT.java

```diff
@@ -65,15 +65,6 @@ public void typeof_opensearch_types() {
                 TEST_INDEX_DATATYPE_NONNUMERIC));
     verifyDataRows(
         response,
-        rows(
-            "TEXT",
-            "TIMESTAMP",
-            "TIMESTAMP",
-            "BOOLEAN",
-            "OBJECT",
-            "KEYWORD",
-            "IP",
-            "BINARY",
-            "GEO_POINT"));
+        rows("TEXT", "DATE", "DATE", "BOOLEAN", "OBJECT", "KEYWORD", "IP", "BINARY", "GEO_POINT"));
```

Comment:
@dai-chen @rupal-bq do u rememeber why we use timestamp for JDBC type?

Reply:
I found some context here: https://github.com/opendistro-for-elasticsearch/sql/pull/716

Reply:
I reverted these changes and IT looks good. We don't need this here.